### ISSUE TITLE
Blue markers

### ISSUE THREAD

**Issue by Gregwar** [RoboCup-SSL/ssl-vision#103]
Hello,

This reference is designated as blue marker in the wiki:
http://www.theroyalstore.com/dark-blue-poster-boards-size-14-in-22-in-p-11165.html

This is what we bought, however it looks really darker than what we can see in the videos for instance. Maybe this is just a feeling, but is it true?

**Comment by g3force** [comment]
we have noticed in previous years, that the colors are not reliable. Indeed, some have received much too dark blue paper. This will probably not work.

We will provide paper for all teams at RoboCup. For your own field, I recommend to choose another blue.

**Comment by Gregwar** [comment]
To be honest we really feel frustrated about this, we spent more than 300$ in shipping to get the exact papers in the wiki because it was only available in US stores, and it is actually a bad/not reliable reference?

Can you indicate us the correct one in this case, which blue should we use?

Are the others correct ?
Our yellow green and pink are really flashy to the camera, we can track them with a really low exposure for instance (with higher it appear white)

**Comment by g3force** [comment]
I'm sorry for this. We haven't come to a conclusion until recently, regarding this decision. I think, we used all but the blue color in previous RoboCups.

It turned out to be quite challenging to find good paper that is available around the world. That's why we want to provide the paper through LOC this year again.

To be honest, I do not like the standard paper that much. We had better experience with bright neon colors. But we got them from a local store, so I have no reference... I suggest to go to your next local store and buy what looks best.

We will put information on the paper into the next mail, which will come soon. And I will update the Wiki.

**Comment by g3force** [comment]
you definitely should be ready to change the patterns during competition. You can use whatever colors you like with your own field, then exchange the patterns during the competition day. And keep them for your field, if you like.

We do not protect our blobs. They got a bit dirty over the time, but I haven't noticed any issues with this yet.

**Comment by g3force** [comment]
I got feedback from other teams: Green and magenta are ok (they are Fluorescent Neon already) and yellow is not that critical, so it should be fine, too. I updated the link to the blue paper to one with a light blue side that should work well, based on the feedback.

A note on the calibration: You want to minimize exposure times to minimize motion blur and latency, and you want to minimize the aperture to maximize the depth of focus, else not all parts of the field will be in focus.
